### DISCUSSION TITLE
My dreams annotation tool 😍

### DISCUSSION THREAD

**Discussion by elifbeyzatok00** [tecoholic/ner-annotator#118] [General]
I am grateful to you to developed this application. Before I found this application, I was doing hours of NER model labeling manually. You saved my life. This app is annotation too of my dreams. Easy to use and functional. 😍

**Comment by tecoholic** [comment]
@elifbeyzatok00 Thank you for the kind words. A lot of the credit goes to @alvi-khan for maintaining the tool.

@alvi-khan Thank you for keeping the tool alive.

**Reply by alvi-khan** [reply]
@elifbeyzatok00 It's always great to hear that people are finding the tool useful! It makes all the effort worthwhile.

@tecoholic I'm grateful for your appreciation, but also humbled by it. After all, this tool wouldn't have existed at all without you. I'm just glad I'm able to play a small part in supporting it.
